src/main/java/uk/gov/hmcts/sptribs/flag/refdata/FlagTypeClient.java

```diff
@@ -3,25 +3,19 @@
 import org.springframework.cloud.openfeign.FeignClient;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RequestHeader;
-import org.springframework.web.bind.annotation.RequestParam;
-import uk.gov.hmcts.ccd.sdk.type.Flags;
-
-import java.util.List;
 
 import static org.springframework.http.HttpHeaders.AUTHORIZATION;
 import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
 import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
-import static uk.gov.hmcts.sptribs.common.CommonConstants.CATEGORY_ID;
 import static uk.gov.hmcts.sptribs.common.config.ControllerConstants.SERVICE_AUTHORIZATION;
 
 @FeignClient(name = "flag-type-client", url = "${refdata_common.api.baseUrl}")
 public interface FlagTypeClient {
-    @GetMapping(value = "/refdata/commondata/caseflags",
+    @GetMapping(value = "/refdata/commondata/caseflags/service-id=sptribs_case_api",
         headers = CONTENT_TYPE + "=" + APPLICATION_JSON_VALUE)
-    List<Flags> getFlags(
+    Object getFlags(
```

Comment:
Does it have to be Object? Ideally, it would be more specifically typed.

Reply:
I was just trying to understand response type, not used, I can remove